src/IIIFPresentation/API/Attributes/EtagCachingAttribute.cs

```diff
@@ -66,7 +65,7 @@ ResultExecutionDelegate next
 
                 // Remove all unnecessary headers while only keeping the ones that should be included in a `304` response.
                 foreach (var header in response.Headers)
-                    if (!headersToKeepFor304.Contains(header.Key))
+                    if (!HeadersToKeepFor304.Contains(header.Key))
```

Comment:
Should the operations within this `foreach` be enclosed in a bracket, following the style used in Protagonist?